themes/theme-ios11/extension-config.json

```diff
@@ -229,14 +229,22 @@
         "colors": {
           "background": "#f8f8f8",
           "light": "#fff",
+          "lightTransparent": "rgba(255, 255, 255, 0.82)",
           "dark": "#000",
+          "lightDark": "#323232",
           "placeholder": "#f2f2f2",
+          "overlay": "#fafafa",
+          "lightOverlay": "#fafafa",
+          "darkOverlay": "#f0f0f0",
           "primary": "#fa5400",
           "primaryContrast": "#fff",
           "accent": "#5ccee3",
           "accentContrast": "#fff",
           "cta": "#fa5400",
           "ctaContrast": "#fff",
+          "gray": "#747474",
+          "lightGray": "#9a9a9a",
```

Comment:
Please respect the comments from `theme-gmd/extension-config.json` here as well!